config/settings.yml

```diff
@@ -15,3 +15,12 @@ replace_commitment_report_path: <%= ENV["REPLACE_COMMITMENT_REPORT_PATH"] %>
 shared_print_report_path: <%= ENV["SHARED_PRINT_REPORT_PATH"] %>
 shared_print_update_report_path: <%= ENV["SHARED_PRINT_UPDATE_REPORT_PATH"] %>
 target_cost: <%= ENV["TARGET_COST"] %>
+solr_data_source:
```

Comment:
I tried to exact all the "magic numbers" to settings, and at least git "batch size" here a prefix.